### PR TITLE
Dev 1130 Add recovery email status in the dashboard page

### PR DESCRIPTION
**Pull request by ahsinsiddique**

### CODE REVIEW

File: src/app/mail/mail-settings/mail-settings.component.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class MailSettingsComponent implements OnInit, AfterViewInit, OnDestroy {
 
   timeZoneFilter = new FormControl('', []);
   timeZoneFilteredOptions: Observable<Timezone[] | void>;
-
+  isRecoveryEmailEdit: boolean;
```
Comment (atifsaddique211f) [inline src/app/mail/mail-settings/mail-settings.component.ts]: rename `isRecoveryEmailEdit ` to `isEditingRecoveryEmail`

File: src/app/mail/mail-settings/mail-settings.component.ts [hunk 2]
```diff
@@ -138,6 +138,9 @@ export class MailSettingsComponent implements OnInit, AfterViewInit, OnDestroy {
   private _filterTimeZone(name) {
     return this.timezones.filter(option => option.value.toLowerCase().indexOf(name.toLowerCase()) > -1);
   }
+  private toggleRecoveyEmailEdit() {
```
Comment (atifsaddique211f) [inline src/app/mail/mail-settings/mail-settings.component.ts]: why this function is private,
you don't need this function.

File: src/app/mail/mail-settings/mail-settings.component.ts (hunk 1 above)
Comment (ahsinsiddique) [reply src/app/mail/mail-settings/mail-settings.component.ts]: fixed typos

File: src/app/mail/mail-settings/mail-settings.component.ts (hunk 2 above)
Comment (ahsinsiddique) [reply src/app/mail/mail-settings/mail-settings.component.ts]: this function is called from multiple buttons, so I have added,
and removed the private identifier.

**Pull request merged**